index.html

```diff
@@ -29,226 +29,257 @@
             </div>
         </header>
         <section class="header">
-            <h1>I'm Anold. <br>Glad to see <br> you!</h1>
-            <p>I'm a software developer! I can help you build a product , feature or website Look through some of my
-                work and experience! If you like what you see and have a project you need coded, don't hesitate to
-                contact me.</p>
-            <article class="social">
-                <a href="#">LET'S CONNECT</a>
-                <ul class="social_icons">
-                    <li>
-                        <a href="#"><img src="images/twitter.svg" alt="twitter icon"></a>
-                    </li>
-                    <li>
-                        <a href="#"><img src="images/Linkedin icon-1.svg" alt="Linkedin icon"></a>
-                    </li>
-                    <li>
-                        <a href="#"><img src="images/medium.svg" alt="medium icon"></a>
-                    </li>
-                    <li>
-                        <a href="#"><img src="images/github.svg" alt="github icon"></a>
-                    </li>
-                    <li>
-                        <a href="#"><img src="images/angellist.svg" alt="angellist icon"></a>
-                    </li>
-                </ul>
-            </article>
-        </section>
-        <section class="work_section">
-            <article class="work">
-                <img src="images/Snapshoot Portfolio4.png" alt="">
-                <h2>Tonic</h2>
-                <div class="canopy">
-                    <div>
-                        <h5>CANOPY</h5>
-                    </div>
-                    <ul class="canopy_languages">
+            <div class="header_container">
```

Comment:
Great work , but you missed the navigation for desktop as you see in the screenshot we still see the hamberger menu on desktop size , kindly add the missing elements

![Capture d’écran du 2023-07-13 12-01-26](https://github.com/MPDADDY/my_portfolio/assets/59707859/0898738b-105d-4ca4-b138-f01c3abbe41c)
![Capture d’écran du 2023-07-13 12-00-21](https://github.com/MPDADDY/my_portfolio/assets/59707859/0792b116-d301-4258-acc2-135755c79de3)